ffi/src/handle.rs

```diff
@@ -165,6 +165,14 @@ mod private {
         pub unsafe fn drop_handle(self) {
             drop(self.into_inner())
         }
+
+        /// In testing code we want to simulate what c code can do where a pointer can be used
+        /// without consuming it. This creates a "new" handle just by cloning the underlying
+        /// pointer. This is unsafe! Do not use outside testing code!
```

Comment:
does this mean we should make it literally `unsafe`?

Reply:
no, `unsafe` as a keyword means it calls unsafe code, this is "safe" in that sense. I will find a better word to use in the comment